utilities/dead_links/dead_link_tally.py

```diff
@@ -14,7 +14,7 @@
 from tqdm import tqdm
 
 
-redis = Redis("localhost", decode_responses=True)
+redis = Redis("localhost", port=6399, decode_responses=True)
```

Comment:
Really should be configurable. I don't like the idea of encoding this to always and specifically read from the default port we map the prod redis to :frowning:. The script is fine in its current implementation, but if it changed for some reason, we shouldn't be using _production redis_ to test and iterate on those changes.

Can you either: revert this from this PR and open a separate issue to make this configurable as needed (preferably using argument parser for CLI args), or else go ahead and make that change in this PR?

To make it configurable, it should be enough to create an ArgumentParser with `--host` and `--port` arguments that default to `localhost` and `None`, and then use those. Don't even need to wrap this in `def main()`, and `if __name__ ...` if you want to keep the line change low.

I think it would be better to put it in a separate PR though.